README.md

```diff
@@ -2,22 +2,6 @@
 
 > Framework for accessing data resources, mapping data models, describing the data to ontologies and perform data transformations
 
-## Content
```

Comment:
What is the reason to remove the content? I think it is useful for people viewing the project on GitHub - which is the entrypoint for most people - even though it is a small burden to keep updated...

Reply:
Since there aren't a lot of things on the main page/README and I'd rather have people use the documentation, I chose to remove it as it is redundant in the documentation. The Table of Contents is already available on the right side in the documentation theme. If you feel it's absolutely necessary I can return it, but then I'd feel forced to try and dynamically remove it when creating the `index.md` for the documentation.
On the other hand, we can add a sentence stating the "we highly recommend people to read this README through [the documentation](https://EMMC-ASBL.github.com/oteapi-core)"?

Reply:
I have added this suggested sentence to the README.